FlowCrypt/Functionality/Services/Backup Services/BackupService.swift

```diff
@@ -11,16 +11,16 @@ import UIKit
 final class BackupService {
     let backupProvider: BackupProvider
     let core: Core
-    let messageSender: MessageGateway
+    let messageGateway: MessageGateway
 
     init(
         backupProvider: BackupProvider,
         core: Core = .shared,
-        messageSender: MessageGateway
+        messageGateway: MessageGateway
     ) {
         self.backupProvider = backupProvider
         self.core = core
-        self.messageSender = messageSender
+        self.messageGateway = messageGateway
```

Comment:
Same question here - how is message sender determined going forward (it may be different than currently logged in account)

Reply:
answered